File: 1-easy/problems/1313-decompress-run-length-encoded-list/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def decompressRLElist(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        return [nums[i+1] for i in range(0, len(nums), 2) for _ in range(nums[i])]

File: 1-easy/problems/1313-decompress-run-length-encoded-list/solution.ts
function decompressRLElist(nums: any): boolean | number | string | any {
  """
  :type nums: List[int]
  :rtype: List[int]
  """
  return [nums[i+1] for i in Array.from({length: 0, (nums}, (_, i) => i).length, 2) for _ in Array.from({length: nums[i]}, (_, i) => i)];
}

export { decompressRLElist };
